android/build_boinc_arm.sh

```diff
@@ -38,16 +38,26 @@ if [ -n "$COMPILEBOINC" ]; then
     echo "===== building BOINC for arm from $BOINC ====="
     cd "$BOINC"
     if [ -n "$MAKECLEAN" ] && [ -f "Makefile" ]; then
-        make distclean 1>$STDOUT_TARGET 2>&1
+        if [ -z "$VERBOSE" ] || [ "$VERBOSE" = false ]; then
+            make distclean --silent 1>$STDOUT_TARGET 2>&1
```

Comment:
Yes, you can extend `build_all.sh` script with `--silent` flag

Reply:
I don't want to. I don't like the --silent flag, because I don't see the progress there.

Reply:
Watching this from the outside - and congratulations to all on the progress you're making - I don't like to see "I want" on a design decision. Surely, "We want" ...

... to enable a consistent set of options, so that each developer can see the level of detail they, individually, want?

Reply:
Please make another PR with better design to fit all the dev. I will not offend.

Reply:
Sorry, I'm not up to speed on the tools you're using - I would hold you back. It was just a suggestion for the team, as a whole, to consider.

Reply:
I am using this tool a lot and I want to be comfortable by settings default parameters, and  only if you have a bug, to change it to VERBOSE=true.
IF you forcing me to use the flag --silent by default, that mean to use /dev/null that mean I will not see the progress of the script. I will prefer to close this pr and not make the change.
Because the settings now is better then with the --silent flag.
Is it more clear and strong reason?

Reply:
Scripts (especially chained) should have similar design for easier usage.
So you can add `--silent` parameter to every script and define (if not defined before) `STDOUT_TARGET`